www/m/index.js

```diff
@@ -1091,6 +1090,34 @@ function updateInfo() {
 	});
 }
 
+function updateEntropy() {
+	r2.cmd ("p=", function(d) {
```

Comment:
It is better to use `p=ej` the way to use this command is the following:

`r2.cmd('p=ej', (x) => ...)`

the argument for `p=ej` specifies in how many blocks you want to split the entire area analyzed. to specify the length just use the second parameter and run this command in a different offset. So the correct command will be:

`p=ej 20 $s @ $M`

note that '20' is the number of rows you will get in the resulting JSON.